aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/data/AWSCognitoLegacyCredentialStore.kt

```diff
@@ -174,8 +182,6 @@ internal class AWSCognitoLegacyCredentialStore(
         val keys = getTokenKeys()
         val cognitoUserPoolTokens = retrieveCognitoUserPoolTokens(keys) ?: return null
         val signInMethod = retrieveUserPoolSignInMethod() ?: return null
-        val username = keys[APP_LAST_AUTH_USER]?.let { tokensKeyValue.get(it) }
```

Comment:
This is required for legacy credentials migration. Please revert.

Reply:
I did not touch this and its actually present in my code. For some reason when it plans to merge it github is removing this.

Reply:
Actually the reason why I removed it was that it is no longer used as we removed it from the signedinData.